src/Tickets/Commerce/Order_Modifiers/Modifiers/Modifier_Abstract.php

```diff
@@ -680,6 +631,59 @@ public function map_form_data_to_model( array $raw_data ): array {
 		];
 	}
 
+	/**
+	 * Handle dynamic method calls.
+	 *
+	 * This should be used for deprecated methods that are no longer in use.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $name      The method name.
+	 * @param array  $arguments The method arguments.
+	 *
+	 * @return mixed The results of the method call.
+	 * @throws InvalidArgumentException If the method does not exist.
+	 */
+	public function __call( $name, $arguments ) {
+		$method = __CLASS__ . "::{$name}";
+		switch ( $name ) {
+			case 'convert_from_raw_amount':
+				_deprecated_function( esc_html( $method ), 'TBD', 'No replacement available.' );
+
+				$amount       = ( $arguments[0] ?? 0 ) / 100;
+				$amount_value = Value::create( $amount );
+
+				return number_format( $amount_value->get_decimal(), 2, '.', '' );
+
+			default:
+				throw new InvalidArgumentException( sprintf( 'Method %s does not exist.', esc_html( $method ) ) );
+		}
+	}
```

Comment:
I'm not familiar with replacing deprecated methods with a `__call()` method. How does this affect our automated docs?

Reply:
@codingmusician I'm not sure. How would I go about checking? I haven't looked into automated docs at all.

Reply:
@Camwyn or @bordoni might know better. This approach may be just fine. I just want to make sure.